core/src/main/resources/templates/editTopicRequest.html

```diff
@@ -434,7 +434,8 @@ <h4 class="mb-3">Shortcuts</h4>
 				<div class="ribbon-wrapper card col-lg-6 col-md-6 col-xlg-2 col-xs-6">
 					<div class="ribbon ribbon-success">New user interface available</div>
 					<p class="ribbon-content">
-						Check out the new interface for <a href="coral/topics/request">topic request.</a>
+						Check out the new interface for <a
+							href="coral/topic/{{ addTopic.topicname }}/request-update?env={{ addTopic.envId }}">edit topic.</a>
```

Comment:
This "editTopicRequest.html" is for editing an existing TopicRequest which is a new feature we havent got screens for in coral yet so we should remove it here,

I believe the place we need to add this into is "requestTopic.html" and add something it like (also it seems to be set as envName instead of envId)
![image](https://github.com/Aiven-Open/klaw/assets/121855584/b5e777dd-7c9a-46fd-a0e7-5956c081a6aa)

Reply:
oooooooooooooh, ok! I've to check again how I got there 🤔  

But linking to `/coral/topic/request` here is also wrong, right? Because as a user I would expect to get the same UI (editing a request) but would get the request overview.

Reply:
/coral/topic/request works for new Topic Requests and the other link works for edit I think the links are right.
New Topic
![image](https://github.com/Aiven-Open/klaw/assets/121855584/b6db4924-1842-4efb-bfe0-e7793abc223f)

Edit topic
![image](https://github.com/Aiven-Open/klaw/assets/121855584/9111b82a-ec1e-4c0a-8857-36e1a86d0275)

Reply:
OK, so the link to edit has to go into `requestTopics.html` and `editTopicRequest.html` should not have a link at all because in coral you can't edit a topic _request_, right? 

Also I just found the view for `editTopicRequest` and I think the headline is wrong? Should I change that to "Topic Edit Request"? 

<img width="504" alt="Screenshot 2023-08-17 at 12 24 58" src="https://github.com/Aiven-Open/klaw/assets/943800/814a7c28-91ca-4143-822b-cd79c203c2eb">

Reply:
- OK, so the link to edit has to go into requestTopics.html and editTopicRequest.html should not have a link at all because in coral you can't edit a topic request, right?
Yeah 100% 



- Also I just found the view for editTopicRequest and I think the headline is wrong? Should I change that to "Topic Edit Request"?
I somehow missed that despite staring at it :D But yes if you can update that to "Edit Request" or "Edit Topic Request" I think?